central/notifiers/awssh/notifier.go

```diff
@@ -166,10 +166,6 @@ func (c *configuration) getCredentials() (*storage.AWSSecurityHub_Credentials, e
 		return c.descriptor.GetAwsSecurityHub().GetCredentials(), nil
 	}
 
-	if c.descriptor.GetNotifierSecret() == "" {
```

Comment:
I'm just recording for posterity.

AWSSH is possible to have empty creds if STS is enabled: https://github.com/stackrox/stackrox/blob/master/central/notifiers/awssh/notifier.go#L142

So good to remove